certified-connectors/Witivio/apiProperties.json

```diff
@@ -1,25 +1,24 @@
 {
```

Comment:
@tdechanterac The apiProperties.json file is outdated one. Please provide the latest one you have uploaded in the ISV Studio.

`{
    "properties": {
        "capabilities": [
            "actions",
            "triggers"
        ],
        "connectionParameters": {
            "token": {
                "oAuthSettings": {
                    "clientId": "9275fab6-86e0-4fb6-affa-1473100e8f22",
                    "customParameters": {
                        "loginUri": {
                            "value": "https://login.windows.net"
                        },
                        "resourceUri": {
                            "value": "https://witivio.com/internalapi"
                        },
                        "tenantId": {
                            "value": "common"
                        }
                    },
                    "identityProvider": "aad",
                    "properties": {
                        "AzureActiveDirectoryResourceId": "https://witivio.com/internalapi",
                        "IsFirstParty": "False"
                    },
                    "redirectMode": "Global",
                    "redirectUrl": "https://global.consent.azure-apim.net/redirect",
                    "scopes": [
                    ]
                },
                "type": "oauthSetting"
            },
            "token:TenantId": {
                "metadata": {
                    "sourceType": "AzureActiveDirectoryTenant"
                },
                "type": "string",
                "uiDefinition": {
                    "constraints": {
                        "hidden": "true",
                        "required": "false"
                    }
                }
            }
        },
        "iconBrandColor": "#8F3163",
        "publisher": "Witivio",
        "stackOwner": "Witivio"
    }
}`